airbyte-webapp/src/components/connection/UpdateConnectionDataResidency/UpdateConnectionDataResidency.tsx

```diff
@@ -50,11 +50,16 @@ export const UpdateConnectionDataResidency: React.FC = () => {
               <FormattedMessage
                 id="connection.geographyDescription"
                 values={{
-                  lnk: (node: React.ReactNode) => (
+                  lnk1: (node: React.ReactNode) => (
```

Comment:
Nitpick, but could we somehow give those tags a bit more meaningfull names :D like `<ipLink>` and `<docLink>`?

Reply:
Sure, that makes sense!